src/Libraries/RevitNodes/Filter/FilterRule.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using Autodesk.Revit.DB;
+using DynamoServices;
+using Autodesk.DesignScript.Runtime;
+using Revit.GeometryConversion;
+using RevitServices.Persistence;
+using RevitServices.Transactions;
+using System.Collections.Generic;
+
+namespace Revit.Filter
+{
+    /// <summary>
+    /// Revit Filter Rule
+    /// </summary>
+    public class FilterRule
+    {
+
+        #region Internal Properties
+
+        /// <summary>
+        /// Internal reference to the Revit Element
+        /// </summary>
+        internal Autodesk.Revit.DB.FilterRule InternalFilterRule
+        { 
+            get; set;
+        }
+
+        /// <summary>
+        /// Reference to the Element
```

Comment:
@mjkkirschner it is a revit element (Autodesk.Revit.DB.FilterRule) but it doesn't require any bindings. I followed similar existing implementations like Parameters (https://github.com/moethu/DynamoRevit/blob/Revit2015/src/Libraries/RevitNodes/Elements/Parameter.cs) So its basically a wrapper for the FilterRule with an internal property containing the actual Revit element.